backend/djangoindia/db/migrations/0008_remove_eventregistration_occupation_and_more.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 4.2.5 on 2024-08-28 17:35
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0007_sponsor_sponsorship'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='eventregistration',
+            name='occupation',
+        ),
+        migrations.AddField(
+            model_name='eventregistration',
+            name='company',
+            field=models.CharField(blank=True, max_length=100, null=True),
+        ),
+        migrations.AddField(
+            model_name='eventregistration',
+            name='description',
+            field=models.TextField(blank=True, null=True),
+        ),
+        migrations.AddField(
+            model_name='eventregistration',
+            name='professional_status',
+            field=models.CharField(choices=[('working_professional', 'working_professional'), ('student', 'student'), ('freelancer', 'freelancer'), ('other', 'other')], default='other', max_length=100),
+        ),
+        migrations.AlterField(
+            model_name='eventregistration',
+            name='gender',
+            field=models.CharField(choices=[('male', 'male'), ('female', 'female'), ('other', 'other')], max_length=15),
+        ),
+        migrations.AlterField(
+            model_name='eventregistration',
+            name='github',
+            field=models.URLField(blank=True, null=True),
```

Comment:
Does this include URL validations? I'm not sure, but if not, we can use the validator package (https://validators.readthedocs.io/en/latest/)

Reply:
yep we have some inbuilt [validators in django](https://github.com/django/django/blob/main/django/core/validators.py#L127) which are enough in our case i guess.